src/domain/accounts/services/src/login_password_auth_provider.rs

```diff
@@ -88,10 +88,14 @@ impl AuthenticationProvider for LoginPasswordAuthProvider {
         &self,
         login_credentials_json: String,
     ) -> Result<ProviderLoginResponse, ProviderLoginError> {
+        dbg!("@@@ login 1", &login_credentials_json);
```

Comment:
If we want some logs to remain, use `tracing`

Reply:
oh, it wasn't supposed to be commited...